### DISCUSSION TITLE
select text without exiting ruler mode

### DISCUSSION THREAD

**Discussion by ramithuh** [ahrm/sioyek#616] [Q&A] [answered]
I'm currently activating ruler mode by `right clink -> down arrow`.

But if I select a text using left mouse, ruler gets disabled. Is there a way to prevent that?

**Comment by ahrm** [comment, marked as answer]
No, but you can return to your last selected line by pressing `` (pressing ` twice).

**Reply by ramithuh** [reply]
Thank you very much for the quick clarification! ❤️

**Reply by ramithuh** [reply]
Hi, I was wondering whether I can override the shortcut ` -> .

i added the following, but not sure whether I can figure out the `last selected line` command
`goto_mark .`

**Reply by ahrm** [reply]
\` is a special symbol which marks the last highlighted line. So when you press \`\`, you are executing goto_mark(\`). So if you remap `goto_mark` to `.`, then you still have to press .\` to go to the last highlighted line. If you want to remap going to the line as a command, you can do something like this:

in prefs_user.config:

```
new_macro _highlight_line goto_mark(`)

```

in keys_user.config

```
_highlight_line .
```

**Reply by ramithuh** [reply]
Thank you so much! it works nicely!
